opendbc/car/hyundai/interface.py

```diff
@@ -26,7 +26,11 @@ def _get_params(ret: structs.CarParams, candidate, fingerprint, car_fw, experime
 
     if ret.flags & HyundaiFlags.CANFD:
       # Shared configuration for CAN-FD cars
-      ret.experimentalLongitudinalAvailable = candidate not in (CANFD_UNSUPPORTED_LONGITUDINAL_CAR | CANFD_RADAR_SCC_CAR)
+      hda2_longitudinal = hda2 and Ecu.adas in [fw.ecu for fw in car_fw] and \
+                          candidate not in CANFD_UNSUPPORTED_LONGITUDINAL_CAR
+
+      ret.experimentalLongitudinalAvailable = hda2_longitudinal or \
+                                              candidate not in (CANFD_UNSUPPORTED_LONGITUDINAL_CAR | CANFD_RADAR_SCC_CAR)
```

Comment:
I don't see how this works? It's supposed to remove support for some cars, but this adds an `or` to the existing condition